docs/admin/integration/spark.md

```diff
@@ -83,6 +83,15 @@ To schedule the executors on GPUs, add the following flags:
 --conf spark.executor.resource.gpu.discoveryScript=/opt/spark/examples/src/main/scripts/getGpusResources.sh \
 ```
 
+With GPU fractions add the annotaiton to the executor pods:
+
+```
+--conf spark.kubernetes.executor.annotation.gpu-fraction=0.5 \
```

Comment:
@enoodle Is there a command that preceedes the option flag `--conf`? I had asked that back on line 69. Is the command `runai-scheduler`? If so we should add it to the lines where the flags are.